### PR TITLE
copy when click on title of terminal in docs

### PR DESCRIPTION
**Pull request by grnnja**
## Describe your changes

Added the title of the terminal to the clickable area for copying terminal text in the docs.

## Issue ticket number and link
Closes https://github.com/ploomber/projects/issues/51

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests (when necessary).
- [x] I have added the right documentation (when needed). Product update? If yes, write one line about this update.

### CODE REVIEW

File: doc/_static/js/custom.js [hunk 1]
```diff
@@ -26,17 +26,26 @@ $(document).ready(function () {
     // TODO: this should listen for click events in the whole terminal
     // window, not only in the content sub-window
     // Clicking on the terminnals copies the content to the clipboard
-    $("div.highlight").click(function () {
-        if ("jquery: ", $(this).parent().attr('class') == "doctest highlight-default notranslate") {
-            let text = $(this).text();
+    $("[class*='highlight-']") .click(function () {
+        var formatCopiedText = function(text) {
```
Comment (idomic) [inline doc/_static/js/custom.js:30]: please change to let, var usage isn't recommended after ES 5 or 6 I think

**Review by idomic — CHANGES_REQUESTED** [review]
please fix

**Comment by idomic** [comment]
@yafimvo can you please review as well?

File: doc/_static/js/custom.js (hunk 1 above)
Comment (grnnja) [reply doc/_static/js/custom.js]: fixed

**Review by yafimvo — COMMENTED** [review]
`if ("jquery: ", $(this).attr('class') == "doctest highlight-default notranslate")`

1. When type coercion is not required, it is recommended to use '===' instead of '=='
2. It may not be necessary to use the comma operator here
3. Elements' class names may change in the future so this `doctest highlight-default notranslate` string format may be different and the comparison will fail. You may avoid this by comparing substrings.

**Comment by grnnja** [comment]
Good points I will fix this. Though it is kinda hard to tell from the diff I didn't actually write this code. I moved some other code but it shows in the diff that the `if` was moved. @idomic am I typically responsible for other people's code?

**Comment by idomic** [comment]
@grnnja I'm not sure how it got there, but let's take the opportunity to fix that :)
It's just a better practice to improve bit by bit and overall in the long run (instead of copy pasting, cleaning a bit etc).

**Comment by idomic** [comment]
luckily that's the only place we're using jquery we can simply avoid using it 👍

File: doc/_static/js/custom.js [hunk 2]
```diff
@@ -26,17 +26,30 @@ $(document).ready(function () {
...
+            return text;
         }
+
+        // https://stackoverflow.com/questions/31128855/comparing-ecma6-sets-for-equality
+        const eqSet = (xs, ys) => 
+            xs.size === ys.size &&
+            [...xs].every((x) => ys.has(x));
+
+        const terminalText = $(this).find('div.highlight').text();
+        const isPythonConsole = eqSet(new Set($(this).attr('class').split(' ')),
```
Comment (yafimvo) [inline doc/_static/js/custom.js:44]: isPythonConsole - Are you sure this comparison is ever true? The term 'doctest' didn't turn up anything on my search.

**Review by yafimvo — CHANGES_REQUESTED** [review]

File: doc/_static/js/custom.js [hunk 3]
```diff
@@ -26,17 +26,30 @@ $(document).ready(function () {
...
             text = textArr.join("\n");
             text = text.replaceAll("... ", "").replaceAll(">>> ", "");
-            navigator.clipboard.writeText(text);
-        } else {
-            navigator.clipboard.writeText($(this).text());
+            return text;
         }
+
+        // https://stackoverflow.com/questions/31128855/comparing-ecma6-sets-for-equality
+        const eqSet = (xs, ys) =>
```
[the remark this answers is not in the document]
Comment (grnnja) [reply doc/_static/js/custom.js]: fixed!

File: doc/_static/js/custom.js (hunk 2 above)
Comment (grnnja) [reply doc/_static/js/custom.js:44]: Yeah, like for this page, the terminals have the classes: https://docs.ploomber.io/en/latest/api/_modules/root/ploomber.DAG.html#ploomber.DAG

I also couldn't find it in the repo so I'd assume it's generated by sphinx

**Review by yafimvo — APPROVED** [review]

**Comment by idomic** [comment]
Good job!

**Pull request merged**